### PR TITLE
Attempt to build NanoUI in Travis

### PR DESCRIPTION
**Pull request by neersighted**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -27,12 +31,17 @@ addons:
       - libstdc++6:i386
       - python
       - python-pip
-  
+
+
 install:
+  - rm -rf ~/.nvm && git clone https://github.com/creationix/nvm.git ~/.nvm && (cd ~/.nvm && git checkout `git describe --abbrev=0 --tags`) && source ~/.nvm/nvm.sh && nvm install $NODE_VERSION
```
Comment (YotaXP) [inline .travis.yml]: Should this be a shallow clone with `--depth 1`?  Or is that not possible with the `git describe` thing?

Comment (optimumtact) [inline .travis.yml]: if I understand how this works correctly, they're looking for the most recent tag, so they will need at least the history up to that point, so I'm not sure a bare clone will be acceptable.

Comment (YotaXP) [inline .travis.yml]: Tried cloning it myself.  Looks like the entire repo is only 1MB anyway, so whatever.  :+1:

**Comment by tkdrg** [comment]
Could you squash these commits?

**Comment by optimumtact** [comment]
Also what is the point of this?

**Comment by neersighted** [comment]
Pretty much.

**Comment by neersighted** [comment]
I plan to add a check to make sure Nano is minified, as well, so you don't commit development-mode build assets.

**Pull request merged**